sky/shell/platform/ios/FlutterView.h

```diff
@@ -5,10 +5,14 @@
 #ifndef FLUTTER_FLUTTERVIEW_H_
 #define FLUTTER_FLUTTERVIEW_H_
 
+#include "sky/services/semantics/semantics.mojom.h"
+
 #include <UIKit/UIKit.h>
 
 @interface FlutterView : UIView
 
+- (instancetype)initWithSemantics:(semantics::SemanticsServer*)semantics;
```

Comment:
Is there a reason this needs to be done at init time? Why not have an `addSemantics` message or something that does work outside of `init`.